x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/filestructurefinder/TimestampFormatFinder.java

```diff
@@ -64,31 +64,31 @@
         // The TOMCAT_DATESTAMP format has to come before ISO8601 because it's basically ISO8601 but
         // with a space before the timezone, and because the timezone is optional in ISO8601 it will
         // be recognised as that with the timezone missed off if ISO8601 is checked first
-        new CandidateTimestampFormat("YYYY-MM-dd HH:mm:ss,SSS Z", "yyyy-MM-dd HH:mm:ss,SSS XX",
+        new CandidateTimestampFormat("yyyy-MM-dd HH:mm:ss,SSS Z", "yyyy-MM-dd HH:mm:ss,SSS XX",
```

Comment:
This change and the others like it in this file are not correct.  The first argument is a Joda timestamp format.  The second argument is the corresponding Java time format, but that already has `yyyy` rather than `YYYY`.

Please revert all the changes to files in the `org.elasticsearch.xpack.ml.filestructurefinder` package (including unit tests).  For 6.7 I will update these to make Java time the primary format and Joda a secondary bonus (for Logstash users), but this is not as simple as changing `YYYY` to `yyyy` and I would rather do it as a completely separate PR.